onadata/libs/utils/async_status.py

```diff
@@ -3,14 +3,18 @@
 PENDING = 0
 SUCCESSFUL = 1
 FAILED = 2
+PROGRESS = 3
+RETRY = 4
+STARTED = 5
```

Comment:
There is no big difference  `STARTED` means the job has started and `PROGRESS` (custom task state) is how the task is progressing. `STARTED` is fired when the task is started but is cleared and we require a custom state when updating the task meta field.